spec/acceptance/f5_persistencedestaddr/rest_spec.rb

```diff
@@ -0,0 +1,49 @@
+require 'spec_helper_acceptance'
+
+describe 'f5_persistencedestaddr' do
+  it 'creates and updates f5_persistencedestaddr' do
+    pp=<<-EOS
+    f5_persistencedestaddr { '/Common/dest_addr1':
+       ensure                          => 'present',
+       match_across_pools              => 'enabled',
+       match_across_services           => 'enabled',
+       match_across_virtuals           => 'enabled',
+       hash_algorithm                  => 'carp',
+       mask                            => '255.255.0.0',
+       timeout                         => '180',
```

Comment:
It'd be good to test the type works with actual integers too.

Reply:
Would you be more specific? Which one to use integers?

Reply:
Just in general it'd be nice to confirm in the tests that integer properties do actually work with integers.  Same with booleans.  It'd be nice to check that you can use either 'enabled' or true.

Reply:
@ericzji what @alexjfisher is suggesting is that the acceptance tests should confirm valid inputs should be tested.  In this case, where values can be a word or an integer.  Both should be tested in fairness.